docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,590 @@
+## Text
+
+Therae are three Text components to display and edit text on the screen:
+
+- The [TextLabel](#textLabel) displays a short text string.
+
+- The [TextField](#textField) is to edit a single line text.
+
+- The [TextEditor](#textEditor) is to edit multi-line text.

```

Comment:
The [TextEditor](#textEditor) class provides a control that allows multi-line text editing.